pixart.js

```diff
@@ -1 +1,18 @@
+$(document).ready(function() {
+  $('button').click(function(evt) {
+    evt.preventDefault()
+    console.log("Clicked")
+  });
+});
+$('#Set Color').bind('keypress', function(e) {
+ if (e.which == 13) {
+     console.log('Enter clicked');
+ }
+ });
 
+$('body').append('<div class = "square">' + toAdd + '</div>');
```

Comment:
This is the right idea! The only problem is you want to do this 20 times, instead of just once.